src/data-loader.ts

```
@@ -814,3 +846,128 @@ export async function loadMonthlyUsageData(
+
+		monthly.windowCount++;
+		monthly.totalCost += window.totalCost;
+		monthly.totalTokens += window.inputTokens + window.outputTokens
+			+ window.cacheCreationTokens + window.cacheReadTokens;
+		monthly.windows.push(window);
+	}
+
+	// Apply session limits if provided
+	if (options?.sessionLimit != null) {
+		for (const monthly of monthlyMap.values()) {
+			monthly.sessionLimit = options.sessionLimit;
```

ESLint

[error] Unsafe member access .sessionLimit on an `error` typed value.

(ts/no-unsafe-member-access)